build/nsis.install.nsh

```diff
@@ -19,7 +19,7 @@ Section "Geth" GETH_IDX
 
   # Create start menu launcher
   createDirectory "$SMPROGRAMS\${APPNAME}"
-  createShortCut "$SMPROGRAMS\${APPNAME}\${APPNAME}.lnk" "$INSTDIR\geth.exe" "--fast" "--cache=512"
+  createShortCut "$SMPROGRAMS\${APPNAME}\${APPNAME}.lnk" "$INSTDIR\geth.exe" "--syncmode fast" "--cache=512"
```

Comment:
Could you instead just remove both "--fast" and "--cache"? The former is the default anyway, the latter is way too low for mainnet